README.md

```diff
@@ -8,7 +8,7 @@ Example:
 GET
 ```java
 HttpRequest request = httpRequest(new TargetUrl("abv.bg")).build();
-ChitaHttp chitaClient = new ChitaHttpClient();// in most cases chitaClient should be injected
+ChitaHttp chitaClient = new HttpClient();// in most cases chitaClient should be injected
```

Comment:
the idea is to test the logic after the request was send, not the request sending that is the benefit that we looking for a I think, that's why we have that object HttpClient,. If we will not mock it, then we need just response = request.params().send(), and this will be just wrapper of the HttpConnection and we if we wont to test it we should create another interface and just use it to fire the request or we have to run test server for each test scenario.
That's why the design is like that. Exactly for the mocking of the service. If you have suggestions we will consider then and implement them.